tests/smoke/vendor/k8s.io/api/apps/v1/BUILD.bazel

```diff
@@ -0,0 +1,24 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
```

Comment:
> Why is this package being vendored now?

Because we'd been using `glide-vc --use-lock-file ...`.   I've added another commit to drop the `--use-lock-file` suggestion, rerun the vendoring without the option, and now we're no longer adding this file.